handbook/technical-overview/auth/system/service-provider-manual/intervention-by-code-change.md

```diff
@@ -0,0 +1,10 @@
+# Intervention by code change
+
+If a fault is diagnosed which needs code change then the software support should follow these guidelines:
+
+1. \**\*\*Retrieve the version of the system which is currently on *Prod\*.
```

Comment:
Reason for using \*\* instead of backslash?
Handbook related?

Reply:
I used the prettier formatter and this is what it gave me.